### PR TITLE
Update corefx to rc3-24208-04

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@tmat, @agocke, @VSadov, @joshfree please review.
@mattgertz for approval.

**Comment by VSadov** [comment]
LGTM

File: build/MSBuildToolset/project.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "Microsoft.Build.Targets": "0.1.0-preview-00023-160527",
     "Microsoft.Build": "0.1.0-preview-00023-160527",
     "MSBuild": "0.1.0-preview-00023-160527",
-    "Microsoft.NETCore.TestHost": "1.0.0-rc3-24206-00",
+    "Microsoft.NETCore.TestHost": "1.0.0-rc3-24208-01",
```
Comment (joshfree) [inline build/MSBuildToolset/project.json]: I think this line should be -00
https://github.com/dotnet/versions/blob/1b3784bee6ec0e71f17b889f8a2c507701fd6398/build-info/dotnet/projectk-tfs/release/1.0.0/Latest_Packages.txt#L4

**Comment by cston** [comment]
@joshfree Changed to -00 in #11869.

**Pull request merged**